Sources/AppStoreConnectCLI/Services/Operations/DeleteBetaGroupOperation.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct DeleteBetaGroupOperation: APIOperation {
+    typealias BetaGroup = AppStoreConnect_Swift_SDK.BetaGroup
+
+    struct Options {
+        let betaGroup: BetaGroup
```

Comment:
what do you think of keeping this to a minimum?
```suggestion
        let id: String
```

Reply:
This change can be made, could be useful for potential error reporting if there were errors. But since we're not using it at the moment I'll remove it